packages/yourloops/components/primary-header-bar.tsx

```diff
@@ -129,7 +129,9 @@ function HeaderBar(props: HeaderProps): JSX.Element {
 
   let accountMenu = null;
   if (auth.isLoggedIn()) {
-    const user = auth.user;
+    const user: User = auth.user as User;
+    const name = `${getUserFirstName(user)} ${getUserLastName(user)}`;
```

Comment:
```suggestion
    const name = t("user-name", { firstName: getUserFirstName(user), lastName: getUserLastName(user) });
```
Key used for that case. In some languages, the order is not firstName + lastName, but the opposite.